### PR TITLE
DOC: Fixes to docstrings formatting

### PR DESCRIPTION
**Pull request by realjohnward**
- [ ] closes #xxxx
- [ ] tests added / passed
- [ ] passes `black pandas`
- [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

**Comment by realjohnward** [comment]
@datapythonista Here is the new PR I said I'd make in the PR you requested changes on.

**Comment by realjohnward** [comment]
How do I resolve the failed checks? @datapythonista

File: pandas/io/pytables.py [hunk 1]
```diff
@@ -431,8 +431,9 @@ def _is_metadata_of(group, parent_group):
 class HDFStore:
```
Comment (jbrockmendel) [inline pandas/io/pytables.py]: can you remove this extra newline

File: pandas/tseries/offsets.py [hunk 2]
```diff
@@ -231,18 +230,25 @@ def __add__(date):
         - microsecond
         - nanosecond
 
+        .
+
```
Comment (datapythonista) [inline pandas/tseries/offsets.py]: Can you remove those?

File: pandas/tseries/offsets.py [hunk 3]
```diff
@@ -231,18 +230,25 @@ def __add__(date):
...
     See Also
     --------
-    dateutil.relativedelta.relativedelta
+    dateutil.relativedelta.relativedelta : The relativedelta type is designed
+
+    to be applied to an existing datetime an can replace specific components of
+
+    that datetime, or represents an interval of time.
```
Comment (datapythonista) [inline pandas/tseries/offsets.py]: to continue with the description of the see also, avoid the blank lines and indent the continued description

**Review by datapythonista — COMMENTED** [review]
looks great, thanks for the fixes @realjohnward 

I added couple of comments, but also check the logs of the CI to see what's exactly complaining about

File: pandas/tseries/offsets.py (hunk 3 above)
Comment (realjohnward) [reply pandas/tseries/offsets.py]: Thank you for getting back to me, just committed the changes you requested.

**Review by datapythonista — APPROVED** [review]
Looks good, nice PR @realjohnward, thanks for working on this.

Feel free to keep working on those errors, we have as much as you can want to fix. :)

**Comment by WillAyd** [comment]
Thanks @realjohnward

**Pull request merged**